### PR TITLE
TST: Added tests for HexBuilder class

### PR DESCRIPTION
**Pull request by SimonaStavarache**
TST

### CODE REVIEW

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    @Test
+    void testBuildHex() {
+        HexBuilder newHexBuildObject = HexBuilder.buildHex();
+        assertEquals(newHexBuildObject.getClass(), HexBuilder.class);
+    }
+
+    @Test
+    void testOctWriting() {
+        HexBuilder newHexBuildObject = HexBuilder.buildHex();
+        int[] listInt = {12, 13, 100, 128};
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Use 0x0c, 0x0d, 0x64, 0x80 for the int literals.

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
+package testify.hex;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Expand the wildcards in the imports so they are all spelled out.

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        String result = hexBuilderCapsule.end().hex();
+        assertEquals(expected, result);
+    }
+
+    @Test
+    void testReturnHexString() {
+        HexBuilder newHexBuildObject = HexBuilder.buildHex();
+        assertEquals(newHexBuildObject.hex().getClass(), String.class);
+    }
+}
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Add a trailing newline at the end of the file.

**Review by joe-chacko — CHANGES_REQUESTED** [review]
Changes as discussed:

- [x] Expand imports.
- [x] Use hex int literals where it is next to a hex string in the code.
- [x] newline at end of file

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java (hunk 1 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Solved

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java (hunk 2 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Solved

File: yoko-testify/src/test/java/testify/hex/HexBuilderTest.java (hunk 3 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/hex/HexBuilderTest.java]: Solved

**Review by jakub-pomykala — APPROVED** [review]
LGTM!

**Pull request merged**